libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -124,17 +124,13 @@ func (m *Manager) Apply(pid int) (err error) {
 				}
 				return err
 			}
-			paths[name] = path
+			m.Paths[name] = path
 		}
-		m.Paths = paths
 		return cgroups.EnterPid(m.Paths, pid)
 	}
 
-	paths := make(map[string]string)
+	m.Paths = make(map[string]string)
```

Comment:
Why not just put this line of code before the above `if` so that the duplicated line inside the above `if` can be omitted?